administrator/components/com_users/src/Model/UsersModel.php

```diff
@@ -385,9 +382,13 @@ protected function getListQuery()
                             'a.otpKey',
                             'a.otep',
                             'a.requireReset',
-                        ]
-                    )
-                );
+                        ];
+            if (PluginHelper::isEnabled('multifactorauth')) {
+                $group_by[] = 'mfa.mfaRecords';
+            }
+
+            $query->join('LEFT', '#__user_usergroup_map AS map2 ON map2.user_id = a.id')
+                  ->group($db->quoteName($group_by));
```

Comment:
```suggestion
            ];

            if (PluginHelper::isEnabled('multifactorauth')) {
                $group_by[] = 'mfa.mfaRecords';
            }

            $query->join('LEFT', '#__user_usergroup_map AS map2 ON map2.user_id = a.id')
                ->group($db->quoteName($group_by));
```
Code style: We indent 4 spaces per indent level. We do not align closing brackets for arrays with the opening bracket, we align it with the right indent level. Same for the `->`. We use an empty line before a control structure like if (in most cases).